azure-pipelines.yml

```
@@ -33,20 +33,16 @@ jobs:
     displayName: 'Run linter'
 
 - job: build
-  displayName: 'Build Containers'
+  displayName: 'Build & Push Container'
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
   dependsOn: 'test'
 
   steps:
-  - task: Docker@1
-    displayName: 'Login: Docker Hub'
-
-    inputs:
-      containerregistrytype: 'Container Registry'
-      dockerRegistryEndpoint: 'DockerHub'
-      command: 'login'
-
-  - task: ShellScript@2
-    displayName: 'Build and deploy containers'
-
+  - task: Docker@2
+    displayName: 'Build & Push Container'
     inputs:
-      scriptPath: scripts/deploy-azure.sh
+      containerRegistry: 'DockerHubV2'
```

I believe this is OK now.
```suggestion
      containerRegistry: 'DockerHub'
```